gradle.properties

```diff
@@ -1,8 +1,8 @@
-version=3.26.0
+version=3.27.0
```

Comment:
should not be here I think

Reply:
This comes from rebase with master https://github.com/h2oai/h2o-3/commit/dc3da1985469291989f5af5c9e8a8272a1714f7a . Author of this row is @michalkurka and just because we made target branch a `rel-yau`.... git shows it as diff within my PR as it is not present in release branch. Maybe we should better change target to master... and Michal will port what is needed to release branch. Because now I don't want to search for what should be removed from this PR. wdyt?

Reply:
yes, you should rebase this again on rel-yau and remove those two commits from master that are unrelated to this.

Reply:
@honzasterba I could rebase with particular commit of course but maybe it is not a big deal to do everything in master and later port it to release. It should went smooth and just keep revieving process out of changes from master like we have now.

Reply:
@honzasterba now it looks fine. Once tests are fine I will merge it.